servicetalk-examples/grpc/debugging/src/main/java/io/servicetalk/examples/grpc/debugging/DebuggingClient.java

```diff
@@ -109,6 +111,15 @@
         // AsyncContext.disable();
     }
 
+    /**
+     * If {@link Boolean#TRUE} then all user data in the payload will be logged in addition to the protocol metadata.
+     * If {@link Boolean#FALSE} then protocol operations will be logged, but user data contents will be omitted.
+     *
+     * <p>This implementation uses a constant function to enable or disable logging of user data, your implementation
+     * could selectively choose to log user data based upon application state or context.</p>
```

Comment:
Consider "selectively choose" -> "selectively choose at runtime"

Reply:
Also, consider clarifying that logging user data may leak sensitive content (header values and payload body) to the logs and users should be careful enabling it in production environments.